.github/workflows/build-rule-docs.yaml

```diff
@@ -0,0 +1,54 @@
+#
+# Build Rule Docs
+#
+
+# NOTES:
+# Automatically build rule docs for pull requests
+
+name: Build Rule Docs
+
+on:
+  pull_request:
+    branches: [ main, 'release/*' ]
+    types: [ opened, reopened, synchronize ]
```

Comment:
This trigger should be ok. Not sure about `reopened`, I guess this could run again when a PR is reopened?